lib/chatBot.js

```diff
@@ -712,26 +734,52 @@ ChatBot.prototype._onLogOnResponse = function(res) {
 					sessionID: sessionid,
 					webCookie: cookies
 				}, function(e, api) {
-					if(e) { that.winston.error('ChatBot ' + that.name + ' error getting API key: ' + e); }
+					if(e) { that.winston.error(that.name + '/chatBot.js: error getting API key: ' + e); }
 					else { that.apikey = api; }
-			  });
-			  that.winston.info('ChatBot ' + that.name + ': logged into Steam web');
+			});
+			that.winston.info(that.name + '/chatBot.js: logged into Steam web');
 			}
 		});
+	} else if(res.eresult === 63) {
+		this.winston.warn(this.name + '/chatBot.js: EResult for logon response: 63 (AccountLogonDenied). Please provide the guardCode sent to your email at ' + res.email_domain);
+		//I'm not sure if we might want to use exit codes in the future? Just set exit codes to eresult num plus 100, leaving 1-100 open.
+		process.exit(163);
+	} else { //if any other eresults need extra data, we should separate them as well.
+		var reason = res.eresult+" ("+this._getEResult(res.eresult)+"). "+(evilEResults[res.eresult] || "Please fix the issue and try again.");
+		this.winston.warn(this.name + '/chatBot.js: EResult for logon response: ' +reason);
+		//open steamworks has comment explanations of many eresults.
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/node-steam-chat-bot/steam-chat-bot/file/969545161/issues/source?bid=1980036&fileBranchId=2848073#l751)